system/gluster_volume.py

```diff
@@ -300,13 +308,14 @@ def set_quota(name, directory, value):
     replicas = module.params['replicas']
     transport = module.params['transport']
     myhostname = module.params['host']
-    start_volume = module.boolean(module.params['start_on_create'])
+    start_on_create = module.boolean(module.params['start_on_create'])
     rebalance = module.boolean(module.params['rebalance'])
+    force = module.boolean(module.params['force'])
 
     if not myhostname:
         myhostname = socket.gethostname()
 
-    options = module.params['options']
+    options = module.params['options'] or {}
```

Comment:
we should just make the default {} instead of None above, this would be redundant